mentoring-exercises/src/App.js

```diff
@@ -0,0 +1,33 @@
+import "./App.css";
+import Counter from "./components/counter";
+import GenreSelect from "./components/genreSelect";
+import SearchForm from "./components/searchForm";
+import React, { useState } from "react";
+
+const genres = ["action", "comedy", "terror"];
+
+function App() {
+  const [selectedGenre, setGenreSelected] = useState(genres[0]);
+
+  const onSearch = (value) => {
+    console.log(`Search ${value}`);
+  };
+
+  const onSelectGenre = (selected) => {
+    setGenreSelected(selected);
+  };
+
+  return (
+    <div className="App">
+      <Counter initialValue={0} />
+      <SearchForm onSearch={onSearch} />
```

Comment:
The task for the search form component must have 2 props: Initial search query. Use the value to set the initial value for the input